package.json

```diff
@@ -55,7 +55,8 @@
     "docs": "node scripts/generate-readme.js",
     "simple-integration-test": "jest --maxWorkers=5 simple-suite",
     "complex-integration-test": "jest --maxWorkers=5 integration",
-    "postinstall": "node ./scripts/postinstall.js"
+    "postinstall": "node ./scripts/postinstall.js",
+    "pre-release": "node ./scripts/pre-release.js"
```

Comment:
yep, that's another way to do it. Both npm and travis have a before_publish script support. I think one way or another doesn't matter much.